proto/packages/package_versions_storage.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package lavanet.lava.packages;
+import "common/entryFixation.proto";
+
+option go_package = "github.com/lavanet/lava/x/packages/types";
+
+
+message PackageVersionsStorage {
```

Comment:
the name storage is very confusing, rename this to PackageEntry

Reply:
done